pkg/capabilities/capabilities.go

```diff
@@ -102,7 +102,7 @@ type CallbackExecutable interface {
 	// Request specific configuration is passed in via the request parameter.
 	// A successful response must always return a value. An error is assumed otherwise.
 	// The intent is to make the API explicit.
-	Execute(ctx context.Context, callback chan<- CapabilityResponse, request CapabilityRequest) error
```

Comment:
@ettec Note -- this will require some changes to capabilities in chainlink and also to the OCR3 capability in this repo. I'm surprised the typechecker hasn't caught the latter case.